src/test/regress/expected/citus_non_blocking_split_columnar.out

```diff
@@ -344,6 +344,15 @@ SET citus.shard_replication_factor TO 1;
             ORDER BY 1, 2;
           relname           |                       Constraint                        |                                                         Definition
 ---------------------------------------------------------------------
+ sensors_2020_01_01_8970002 | fkey_from_child_to_child_8970002                        | FOREIGN KEY (eventdatetime, measureid) REFERENCES colocated_partitioned_table_2020_01_01_8970010(eventdatetime, measureid)
```

Comment:
should we disable defer_drop_after_shard_split in some of these tests? otherwise they seem very likely to get flappy

Reply:
Since deferred drop is default on instead of disabling and stopping the code path, I have set the 'defer_shard_delete_interval' to a high value.